download-basket/src/main/java/downloadbasket/data/NormalWayDownloads.java

```diff
@@ -0,0 +1,49 @@
+package downloadbasket.data;
+
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class NormalWayDownloads {
+	private final Logger LOGGER = LogFactory.getLogger(NormalWayDownloads.class);
+	private Set<String> downloads = new HashSet<>();
+
+	/**
+	 * Add normal way download.
+	 * 
+	 * @param download
+	 */
+	public void addDownload(String download) {
+		if (!downloads.contains(download)) {
```

Comment:
Since `downloads` is now a `Set<String>` this if is redundant as `Set` doesn't allow duplicates.